Mathlib/Algebra/BigOperators/Group/Finset.lean

```diff
@@ -2634,3 +2636,16 @@ end AddCommMonoid
 
 @[deprecated (since := "2023-12-23")] alias Equiv.prod_comp' := Fintype.prod_equiv
 @[deprecated (since := "2023-12-23")] alias Equiv.sum_comp' := Fintype.sum_equiv
+
+-- TODO: move
```

Comment:
I think it's okay here?

Reply:
I was hoping the import-countinf bot would run

Reply:
@adomani, do you know why the bot isn't running?

Reply:
Nevermind, resolved!

Reply:
Can we at least get rid of the `TODO` line?

Reply:
```suggestion
```